src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -66,6 +70,13 @@ export class OrganizationSubscriptionComponent implements OnInit {
     this.loading = true;
     this.userOrg = await this.organizationService.get(this.organizationId);
     this.sub = await this.apiService.getOrganizationSubscription(this.organizationId);
+    
+    const orgs = await this.stateService.getOrganizations();
+    const isAllowedOrgType = Object.values(orgs).some(org => org.planProductType === (ProductType.Enterprise | ProductType.Families));
```

Comment:
I believe we could use `this.userOrg` here instead. As `getOrganizations` will fetch all, and we're only interested in the currently selected org.

Reply:
I see that property now - can it be compared to the same enum?

Reply:
I think so.